.storybook/main.ts

```diff
@@ -0,0 +1,47 @@
+import { StorybookConfig } from '@storybook/angular';
+import remarkGfm from 'remark-gfm';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/angular',
+    options: {},
+  },
+  stories: [],
+  addons: [
+    '@storybook/addon-controls',
+    {
+      name: '@storybook/addon-storysource',
+      options: {
+        loaderOptions: {
+          prettierConfig: { printWidth: 80, singleQuote: false },
+        },
+      },
+    },
+    '@storybook/addon-actions',
+    '@storybook/addon-a11y',
+    'storybook-addon-pseudo-states',
+    '@storybook/addon-viewport',
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        mdxPluginOptions: {
+          mdxCompileOptions: {
+            remarkPlugins: [remarkGfm],
```

Comment:
remarkGfm plugin is necessary for the tables in documentation to work: https://github.com/storybookjs/storybook/issues/20334